npcs/_0gc.lua

```diff
@@ -0,0 +1,41 @@
+-----------------------------------
```

Comment:
https://github.com/myera/darkstar/blob/master/npcs/_0gc.lua
Same for all your files..the npc subdir was left in root instead of in the zone when you made the commit.